app/routes/docs.notices.schema.practices.mdx

```diff
@@ -0,0 +1,26 @@
+---
+handle:
+  breadcrumb: Schema Practices
+---
+
+# Best Practices for Schema
+
+### Recommended Unit System for Your Schema
+
+To maintain a consistent database for the same property reported by different instruments, it is best to primarily use International System of Units (SI) or standard units to ensure uniformity across all notices types. One can easily access unit conversion software, such as astropy, for internal team use.
+
+### Choosing between "allOf", "anyOf", or "oneOf"
+
+We recommend using the "allOf" option because none of the core schema properties are specified as required. However, the flexibility of not defining the required properties while using "anyOf" can lead to false positives, as technically, anything will be validated against it.
```

Comment:
```suggestion
JSON Schema supports the use of the keywords `allOf`, `anyOf`, and `oneOf` to combine schemas. These behave as logical AND, OR, and XOR operators. More often than not, if you are considering using existing Core schema elements, you will likely want to use the `allOf` option, as this will enforce that each listed sub-schema element is valid in your schema's structure. In our [schema definintion docs](/docs/notices/schema#define-your-schema) the `sample.schema.json` uses `allOf` to combine the Alert and Localization schemas with a couple additional properties.

It is important to keep in mind that including existing schema elements in your schema design will not override the requirement rules defined on those elements. If a property defined on an included sub-schema item is required in its own definition, it will also be required in your implementations.
```

Reply:
@lpsinger, @Vidushi-GitHub , @jracusin Is this wording clear?